ivy/data_classes/container/experimental/creation.py

```diff
@@ -1200,3 +1200,93 @@ def mel_weight_matrix(
+        simply wraps the function, and so the docstring for ivy.unsorted_segment_mean
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        data
+            input array or container from which to gather the input.
+        segment_ids
+            Must be in the same size with the first dimension of `data`. Has to be
+            of integer data type. The index-th element of `segment_ids` array is
+            the segment identifier for the index-th element of `data`.
+        num_segments
+            An integer or array representing the total number of distinct segment IDs.
+        key_chains
+            The key-chains to apply or not apply the method to. Default is ``None``.
+        to_apply
+            If True, the method will be applied to key_chains, otherwise key_chains
+            will be skipped. Default is ``True``.
+        prune_unapplied
+            Whether to prune key_chains for which the function was not applied.
+            Default is ``False``.
+        map_sequences
+            Whether to also map method to sequences (lists, tuples).
+            Default is ``False``.
+
+        Returns
+        -------
+        ret
+            A container, representing the result of a segmented mean operation.
+            For each segment, it computes the mean of values in `data` where
+            `segment_ids` equals to segment ID.
+        """
+        return ContainerBase.cont_multi_map_in_function(
+            "unsorted_segment_mean",
+            data,
+            segment_ids,
+            num_segments,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+        )
+
+    def unsorted_segment_mean(
+            self: ivy.Container,
+            segment_ids: ivy.Container,
+            num_segments: Union[int, ivy.Container],
+    ) -> ivy.Container:
+        r"""
+        ivy.Container instance method variant of ivy.unsorted_segment_mean. This method
+        simply wraps the function, and so the docstring for ivy.unsorted_segment_mean
+        also applies to this method with minimal changes.
+
+        Parameters
+        ----------
+        self
+            input array or container from which to gather the input.
+        segment_ids
+            Must be in the same size with the first dimension of `self`. Has to be
+            of integer data type. The index-th element of `segment_ids` array is
+            the segment identifier for the index-th element of `self`.
+        num_segments
+            An integer or array representing the total number of distinct segment IDs.
+
+        Returns
+        -------
+        ret
+            A container, representing the result of a segmented mean operation.
+            For each segment, it computes the mean of values in `self` where
+            `segment_ids` equals to segment ID.
+        """
```

Comment:
docstring missing Examples

Reply:
sorry accidentally seemed to have closed this, could you please add examples here?

Reply:
Done.

Reply:
example should be of container here, not ivy.array

Reply:
oh sorry, corrected.

Reply:
![image](https://github.com/unifyai/ivy/assets/52187856/072dae3b-99aa-466e-8dcb-6cfc54758034)
docstring example doesn't work?

Reply:
Hello Rishab,

I ran the example on my local but it showed an error. the implementation seems correct but I don't know why the example not running. Could you please check what the actual cause is? Thank you!